pandas/tests/test_window.py

```diff
@@ -828,74 +814,37 @@ def test_centered_axis_validation(self):
              .rolling(window=3, center=True, axis=2).mean())
 
     def test_rolling_sum(self):
-        self._check_moment_func(mom.rolling_sum, np.sum, name='sum')
+        self._check_moment_func(np.sum, name='sum')
 
     def test_rolling_count(self):
         counter = lambda x: np.isfinite(x).astype(float).sum()
-        self._check_moment_func(mom.rolling_count, counter, name='count',
-                                has_min_periods=False, preserve_nan=False,
+        self._check_moment_func(counter, name='count', has_min_periods=False,
                                 fill_value=0)
 
     def test_rolling_mean(self):
-        self._check_moment_func(mom.rolling_mean, np.mean, name='mean')
+        self._check_moment_func(np.mean, name='mean')
 
     def test_cmov_mean(self):
         # GH 8238
         tm._skip_if_no_scipy()
 
         vals = np.array([6.95, 15.21, 4.72, 9.12, 13.81, 13.49, 16.68, 9.48,
                          10.63, 14.48])
-        xp = np.array([np.nan, np.nan, 9.962, 11.27, 11.564, 12.516, 12.818,
-                       12.952, np.nan, np.nan])
-
-        with catch_warnings(record=True):
-            rs = mom.rolling_mean(vals, 5, center=True)
-            tm.assert_almost_equal(xp, rs)
-
-        xp = Series(rs)
-        rs = Series(vals).rolling(5, center=True).mean()
-        tm.assert_series_equal(xp, rs)
+        result = Series(vals).rolling(5, center=True).mean()
+        expected = Series([np.nan, np.nan, 9.962, 11.27, 11.564, 12.516,
+                           12.818, 12.952, np.nan, np.nan])
+        tm.assert_series_equal(expected, result)
 
     def test_cmov_window(self):
         # GH 8238
         tm._skip_if_no_scipy()
 
         vals = np.array([6.95, 15.21, 4.72, 9.12, 13.81, 13.49, 16.68, 9.48,
                          10.63, 14.48])
-        xp = np.array([np.nan, np.nan, 9.962, 11.27, 11.564, 12.516, 12.818,
-                       12.952, np.nan, np.nan])
-
-        with catch_warnings(record=True):
-            rs = mom.rolling_window(vals, 5, 'boxcar', center=True)
```

Comment:
don’t remove tests just because we are catching the warning

you may need to rewrite them to use the new syntax
unless they duplicate current tests